tools/sphinx_ext/run_examples.py

```diff
@@ -94,7 +93,6 @@ def run() -> None:
             count += 1
         finally:
             proc.kill()
-            AVAILABLE_PORTS.append(port)
 
             port = _get_available_port()
```

Comment:
This can now be moved into the `except StartupError` since we don't need the original port to append to the `AVAILABLE_PORTS` anymore.